app/Services/Adselect.php

```diff
@@ -33,11 +31,11 @@ class Adselect
     public function __construct($endpointUrl, $debug = false)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$debug". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1172)